ui/src/views/Clients/ViewClients/index.jsx

```diff
@@ -36,51 +41,35 @@ import ErrorPanel from '../../../components/ErrorPanel';
 }))
 export default class ViewClients extends PureComponent {
   state = {
-    clientSearch: '',
-    // eslint-disable-next-line react/no-unused-state
-    previousClientId: '',
-    // This needs to be initially null in order for the defaultValue to work
-    value: null,
+    search: '',
```

Comment:
Now that the search value is embedded in the url, it probably doesn't make sense to keep the search value in the state variable.